modules/web-console/frontend/app/services/configuration-checker.js

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
I'd move the file from "services" to "domain" dir. This module is not an AngularJS service.

Reply:
Moved.